src/nerve_tools/configure_nerve.py

```diff
@@ -142,11 +143,22 @@ def generate_subconfiguration(
                         'headers': healthcheck_headers,
                     },
                 ],
-                'labels': labels,
+                'labels': default_labels,
             }
 
-            v2_key = '%s.%s:%d.v2.new' % (
-                service_name, zk_location, port,
+            if typ != discover:
+                continue
+
+            labels = default_labels.copy()
+            if default_labels[typ] != loc:
+                # this is an extra_advertise
+                labels['remote'] = 'true'
```

Comment:
I think we can just have a single "remote" key that holds the remote instance of the type (rename remote_dst_loc to remote)